### PR TITLE
Task/add more default typedefs

### PR DESCRIPTION
**Pull request by jbertel2**
Making sure that contentTypes with no entries have the created_at and created_by fields, which are default fields for entries.

This is required to not break queries against empty stacks / content types without entries

### CODE REVIEW

**Comment by ninadhatkar** [comment]
@jbertel2 thanks for your contribution we will review the pull request and keep you posted.

**Review by ninadhatkar — APPROVED** [review]

**Comment by jbertel2** [comment]
@ninadhatkar  Anything blocking this from being merged?

**Comment by ninadhatkar** [comment]
@jbertel2 We are planning to have this scheduled for next week's sprint where we will go through QA. Will keep you posted.

**Comment by ninadhatkar** [comment]
@jbertel2 We noticed one issue while doing QA on this pull request. The data type isoDate is incorrect and it is an issue with the existing filed as well i.e. updatedAt. We are going to update this as a Date once we merge it and push the change along with this. Can you update this here?

File: src/normalize.js
Comment (ninadhatkar) [inline src/normalize.js:441]:
```suggestion
    data_type: 'Date',
```

**Review by ninadhatkar — CHANGES_REQUESTED** [review]

**Pull request merged**
